### PR TITLE
core: fix input to TrainScheduleV2 into 'Offset<TravelledPath>'

### PR DESCRIPTION
**Pull request by woshilapin**

### CODE REVIEW

File: core/src/main/kotlin/fr/sncf/osrd/standalone_sim/ScheduleMetadataExtractorV2.kt [hunk 1]
```diff
@@ -122,6 +122,7 @@ fun runScheduleMetadataExtractor(
             envelopeAdapter,
             incrementalPath
         )
+
```
Comment (bougue-pe) [inline core/src/main/kotlin/fr/sncf/osrd/standalone_sim/ScheduleMetadataExtractorV2.kt]: nit: :thinking: Not sure this change is necessary (or weird that it's OK with CI before and after)

**Review by bougue-pe — APPROVED** [review]
Can we add a (integration) test for that ?
Or we just let it to the ongoing PR about on_stop_signal handling?

File: core/src/main/kotlin/fr/sncf/osrd/standalone_sim/ScheduleMetadataExtractorV2.kt (hunk 1 above)
Comment (woshilapin) [reply core/src/main/kotlin/fr/sncf/osrd/standalone_sim/ScheduleMetadataExtractorV2.kt]: Yes, I let that slip. I removed it.

**Comment by Khoyo** [comment]
> Can we add a (integration) test for that ?
> Or we just let it to the ongoing PR about on_stop_signal handling?

This doesn't actually change anything except for typing. I'm not sure how you'd test that.

**Review by Khoyo — APPROVED** [review]

**Pull request merged**